lib/PuppeteerSharp.Tests/SingleFileDeployment/PublishedSelfContainedBinary.cs

```diff
@@ -0,0 +1,95 @@
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.SingleFileDeployment
+{
+    public class PublishedSelfContainedBinary
+    {
+        [FactRunableOnWindows]
+        public void PublishedSelfContainedBinaryShouldWork()
+        {
+            var tempPath = Path.GetTempPath();
+            var actualFilePath = Path.Combine(tempPath, $"google.jpg");
+            var actualWindowsBinary = DotnetPublishFolderProfileWindows("PuppeteerSharp.Tests.SingleFileDeployment");
+
+            DeleteIfExists(actualFilePath);
+
+            using var process = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = actualWindowsBinary,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true,
+                    WorkingDirectory = tempPath
+                }
+            };
+
+            process.Start();
+            var outputResult = GetStreamOutput(process.StandardOutput);
+            var errorResult = GetStreamOutput(process.StandardError);
+            var isExited = process.WaitForExit(80000);
+
+            if (!isExited)
+            {
+                process.Kill();
+            }
+
+            Assert.True(isExited);
+
+            Assert.True(File.Exists(actualFilePath), $"StdOut: {outputResult}\nStdErr: {errorResult}\n");
+        }
+
+        private static string GetStreamOutput(StreamReader stream)
+        {
+            var outputReadTask = Task.Run(() => stream.ReadToEnd());
+
+            return outputReadTask.Result;
+        }
+
+        private static string DotnetPublishFolderProfileWindows(string projectName)
+        {
+            var absolutePath = Path.GetFullPath("../../../../" + projectName);
+            var expectedBinaryPath = Path.Combine(absolutePath, $"bin/Release/net6.0/publish/{projectName}.exe");
+
+            DeleteIfExists(expectedBinaryPath);
+
+            using var process = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = "dotnet",
+                    Arguments = $"publish {absolutePath} --configuration Release -f net6.0 -p:PublishProfile=FolderProfile",
```

Comment:
I don't see a [FolderProfile.pubxml](https://github.com/stesee/PuppeteerSharpMinimalRepro/blob/master/PuppeteerSharpMinimalRepro/Properties/PublishProfiles/FolderProfile.pubxml) file in this `PR`. Am I missing something?

For simplicity is it better to just include the args here directly?

```
-r win-x64 -p:PublishSingleFile=true --self-contained false
```

Reply:
`--use-current-runtime` instead of specifying a `RID` would actually be preferable.

Make it easer to take this test cross platform